hydradx/tests/test_omnipool_amm.py

```diff
@@ -1974,3 +1974,283 @@ def test_dynamic_fees_with_trade(liquidity: list[float], lrna: list[float], orac
         expected_asset_fee = min(max(prev_asset_fees[tkn] + df, asset_fee_params['minimum']), asset_fee_params['fee_max'])
         if omnipool.last_fee[tkn] != pytest.approx(expected_asset_fee, rel=1e-15):
             raise AssertionError('Asset fee is not correct.')
+
+
+@given(asset_quantity_strategy, omnipool_config())
+def test_LP_delta_r(lp_amount, omnipool: oamm.OmnipoolState):
```

Comment:
How is this related to invest all?

Reply:
Nothing. I believe these tests are missing from the main branch, but were present in the branch I cloned this branch from. They should be added back, but maybe as part of a different pull request.